priv/repo/migrations/20210429220710_charges_increase_cost_size.exs

```diff
@@ -0,0 +1,15 @@
+defmodule TeslaMate.Repo.Migrations.ChargesIncreaseCostSize do
+  use Ecto.Migration
+
+  def change do
+    alter table(:charging_processes) do
+      modify(:cost, :decimal, precision: 8 , scale: 2, from: :decimal, precision: 6 , scale: 2)
+    end
+  end
+
+  def down do
```

Comment:
This is not necessary if you already defined `change`. You can either define `up` and `down` functions separately or write a [change callback](https://hexdocs.pm/ecto_sql/Ecto.Migration.html?#module-change).